src/frontend/components/LiveVideoPanel/LiveVideoTabPanel/index.spec.tsx

```diff
@@ -0,0 +1,113 @@
+import React from 'react';
+import renderer from 'react-test-renderer';
+import { render, screen } from '@testing-library/react';
+
+import { LivePanelItem } from 'data/stores/useLivePanelState';
+import { wrapInIntlProvider } from 'utils/tests/intl';
+
+import { LiveVideoTabPanel } from '.';
+
+describe('<LiveVideoTabPanel /> titles', () => {
+  it('render for item LivePanelItem.APPLICATION', () => {
+    render(
+      wrapInIntlProvider(
+        <LiveVideoTabPanel item={LivePanelItem.APPLICATION} selected={false} />,
+      ),
+    );
+
+    screen.getByText('application');
+  });
+
+  it('render for item LivePanelItem.CHAT', () => {
+    render(
+      wrapInIntlProvider(
+        <LiveVideoTabPanel item={LivePanelItem.CHAT} selected={false} />,
+      ),
+    );
+
+    screen.getByText('chat');
+  });
+
+  it('render for item LivePanelItem.JOIN_DISCUSSION', () => {
```

Comment:
ditto it

Reply:
https://github.com/openfun/marsha/pull/1248/commits/5ba9c5d50ee394b6339a13fb41c06a8d200457b5